build_sg2_smali_bundle.py

```
@@ -0,0 +1,89 @@
+
+    notes = bundle_root / "PATCH_NOTES.md"
+    notes.write_text(
+        """# SG2 smali 补丁包
+
+这是基于 `三国大时代2一统天下_base.apk` 直接反编译后生成的 smali 补丁文件。
+
+## 已修改
+1. `GTS.smali`
+   - 注入“上帝模式”按钮。
+   - 添加 `toggleGodMode()`。
+   - `AdCheckLoad()` 直接 return，减少运行时广告加载开销。
```

Ruff (0.15.6)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)